### PR TITLE
Reorganize style guides

### PR DESCRIPTION
**Pull request by talsraviv**
Long story short, got rid of style guide section and put any relevant content in appropriate places throughout the docs; then added updated info.

### CODE REVIEW

File: duckduckhack/instant-answer-display/design_styleguide.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```javascript
+DDG.settings.region.getLargeIconURL("CA");
+```
+
+## Spirit of Instant Answers
+
+One guideline that is more abstract is to maintain the spirit of Instant Answers. Instant Answers are meant to first and foremost provide users with value in an accessible way.
+
+Using Instant Answers for other goals, such as excessive self-promotion, or use as an advertisement, at the expense of usability, consistency, or usefulness - is not in line with the spirit of the community.
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/design_styleguide.md]: wondering if we should rephrase this along the lines of "Attribution" and explain how data/source attribution works, meaning the "More at" link, which is a uniform element across all IA's and is used to credit the data provider as well as give the user a link to more information

**Comment by moollaza** [comment]
@talsraviv looks pretty good. Can you verify if we have any links to the styleguides within the rest of the docs?

**Comment by talsraviv** [comment]
@moollaza good catch. Done.

File: duckduckhack/instant-answer-display/design_styleguide.md (hunk 1 above)
Comment (talsraviv) [inline duckduckhack/instant-answer-display/design_styleguide.md]: Good point.

**Comment by talsraviv** [comment]
@moollaza ready to merge?

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**